GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -14,6 +14,7 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text;

```

Comment:
Commenting here to record some of our offline conversation.  

It looks like with these changes users might see files start showing up outside of their included folders if they use `git reset`:

- `git reset --soft`: Will report files outside of the included folders
- `git reset --mixed`: This could result in files outside of the include folders being added to ModifiedPaths.dat

We probably don't need to do anything for this now, but it might be worth mentioned in the release notes and/or having a follow up issue to see if there's anything we can do to improve this experience.